auth-external-api/src/test/java/uk/gov/di/authentication/external/services/UserInfoServiceTest.java

```diff
@@ -144,7 +147,7 @@ private static Stream<Arguments> provideTestData() {
     private static AccessTokenStore getMockAccessTokenStore(List<String> claims) {
         var accessTokenStore = mock(AccessTokenStore.class);
         when(accessTokenStore.getSubjectID()).thenReturn(TEST_SUBJECT.getValue());
-        when(accessTokenStore.getSectorIdentifier()).thenReturn(TEST_RP_SECTOR_URI);
+        when(accessTokenStore.getSectorIdentifier()).thenReturn(TEST_RP_SECTOR_HOST);
```

Comment:
This reflects the real behaviour. This value comes from the access token store, which comes from the auth code store, which comes from the auth frontend request. That is set to the host here:https://github.com/govuk-one-login/authentication-frontend/blob/main/src/components/auth-code/auth-code-service.ts#L44

I've also verified this by checking the access token store dynamo in sandpit